source/_patterns/01-molecules/navigation/subfooter-navigation.twig

```diff
@@ -0,0 +1,11 @@
+{% if subfooter_nav %}
+    <div class="footer__legal-links font--secondary--xs">
+        {% for item in subfooter_nav.items %}
+            <a class="hover link--brown-light {% if not count.last %}space-half--right{% endif %}"
+               href="{{ item.url }}"
+            >
```

Comment:
Should this closing bracket be hanging out here all by itself?

Reply:
I use this when the tag is too long to hold in one line. I put an attribute on each line and close it on line after. I’s an habit from PHP : 
* get no line over 120 chars
* the closing of a list of items must be done on another line, to avoid change on the previous line if you add an item (is that clear ?)

Reply:
Yes, that's clear and seems fine if writing PHP, but just feels a little odd in HTML/Twig. Why not just format it like this? I usually keep native inline elements on one line. 

```twig
{% for item in subfooter_nav.items %}
  <a class="hover link--brown-light {% if not count.last %}space-half--right{% endif %}" href="{{ item.url }}">{{ item.text }}</a>
{% endfor %}
```

There's no character line counting necessary (except maybe for comments).

Reply:
Ok, I take good notice of it. For your information, the 120 limit of PHP PSR-2 standard is specially done for accessibility on any screen, even in a terminal text editor through SSH on a production server with a limited shell :)